doc/deimos.rst

```diff
@@ -12,26 +12,21 @@ settings that are related to the Keck/DEIMOS spectrograph.
 Deviations
 ==========
 
-Here are the deviations from the default settings
-for DEIMOS (set in the settings.keck_deimos file)::
-
-    settings trace slits sigdetect 50.0
-    settings trace slits number -1
-    settings trace slits tilts params 1,1,1
-    settings trace slits tilts method spca
-    settings trace slits pca params [3,2,1,0]
-    settings trace slits polyorder  3
-    settings trace slits sobel mode nearest
-    settings trace slits fracignore  0.02   # 0.02 removes star boxes of 40pix size or less (and any real ones too!)
-    settings bias useframe overscan
-    settings pixelflat combine method median
-    settings pixelflat combine reject level [10.0,10.0]
+Here are the main deviations from the default settings
+for DEIMOS (see `default_pypeit_par`in
+the spectrographs.keck_deimos.py file)::
+
+    # Default lamps
+    par['calibrations']['wavelengths']['lamps'] = ['ArI','NeI','KrI','XeI']
+    # Do not require bias frames
+    turn_off = dict(use_biasimage=False)
+    par.reset_all_processimages_par(**turn_off)
+    # Spectral flexure correction
+    par['flexure']['spec_method'] = 'boxcar'
```

Comment:
Should we do this, or should we point to the pypeitpar docs?  Can't remember at the moment how to do this in rst, but you want it to point here: https://pypeit.readthedocs.io/en/latest/pypeit_par.html#keck-deimos-keck-deimos .

And for any additional changes, instead of giving the python code, should we put this in terms of what needs to change in the pypeit file?

(Side note: We need to make it clear that the changes listed for each instrument on readthedocs are things that pypeit does automatically, not something that a user should include in their pypeit file.  I think I saw someone post a pypeit file to the PypeItUsers slack that basically did this.)

Reply:
Good points!  Am going to defer this to the time when 
we make a real effort to have these instrument-specific
docs be useful..   :)